misk/src/main/kotlin/misk/web/actions/WebActionFactory.kt

```diff
@@ -148,17 +151,27 @@ internal class WebActionFactory @Inject constructor(
     pathPattern: String,
     dispatchMechanism: DispatchMechanism
   ) {
-    // NB: The response media type may be omitted; in this case only generic return types (String,
-    // ByteString, ResponseBody, etc) are supported
-    val action = function.asAction(dispatchMechanism)
-    result += newBoundAction(provider, pathPattern, action)
+    val responseContentTypes = function.findAnnotation<ResponseContentType>()
+      ?.value
+      ?.toList()
+      ?: listOf(null)
```

Comment:
Do we need to set null here so that the flatMap on L159 has something to map over? Or can we just have an empty list?